### PR TITLE
Update benchmark-tidb-using-ch.md

### PR DESCRIPTION
**Pull request by hehechen**
Signed-off-by: hehechen <awd123456sss@gmail.com>

### What is changed, added or deleted? (Required)
Update benchmark-tidb-using-ch.md

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by JaySon-Huang** [comment]
We should replace this `go-tpc` with `tiup bench`
image

**Review by JaySon-Huang — APPROVED** [review]

File: benchmark/benchmark-tidb-using-ch.md [hunk 1]
```diff
@@ -82,8 +82,8 @@ SELECT * FROM information_schema.tiflash_replica WHERE TABLE_SCHEMA = 'tpcc';
 
 查询结果中：
 
-* AVAILABLE 字段表示该表的 TiFlash 副本是否可用。1 代表可用，0 代表不可用。副本状态为可用之后就不再改变，如果通过 DDL 命令修改副本数则会重新计算同步进度。
-* PROGRESS 字段代表同步进度，在 0.0~1.0 之间，1.0 代表至少 1 个副本已经完成同步。
+* `AVAILABLE` 字段表示该表的 TiFlash 副本是否可用。1 代表可用，0 代表不可用。副本状态变为可用之后就不再改变。
+* `PROGRESS` 字段代表同步进度，在 0.0~1.0 之间，1 代表 TiFlash 副本已经完成同步。
```
Comment (qiancai) [inline benchmark/benchmark-tidb-using-ch.md:86]: TiFlash 副本已经完成同步会输出结果会显示为 1 还是 1.0 呢

Comment (hehechen) [reply benchmark/benchmark-tidb-using-ch.md:86]: > TiFlash 副本已经完成同步会输出结果会显示为 1 还是 1.0 呢

会显示1

Comment (qiancai) [inline benchmark/benchmark-tidb-using-ch.md:86]:
```suggestion
* `PROGRESS` 字段代表同步进度，进度值在在 0 到 1 之间，1 代表 TiFlash 副本已经完成同步。
```

**Review by qiancai — APPROVED** [review]

**Pull request merged**